File: T1/proyectos/T01_Repaso/src/utils/Alumno.java
package utils;

public class Alumno {

    private String nombre, apellido, dni;
    private int matricula;

    public Alumno(String nombre, String apellido, String dni, int matricula) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.dni = dni;
        this.matricula = matricula;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getDni() {
        return dni;
    }

    public void setDni(String dni) {
        this.dni = dni;
    }

    public int getMatricula() {
        return matricula;
    }

    public void setMatricula(int matricula) {
        this.matricula = matricula;
    }

    @Override
    public String toString() {
        return "Nombre "+getNombre()+getApellido()+getDni()+getMatricula();
    }
}

File: T7/codigos_ret/angular/academia/src/app/model/alumno.ts
export class Alumno {
  constructor(
    private nombre: string,
    private apellido: string,
    private dni: string,
    private nia: Number,
    private ciclo: string
  ) {}

  public get getNombre(): string {
    return this.nombre;
  }
  public get getApellido(): string {
    return this.apellido;
  }
  public get getDni(): string {
    return this.dni;
  }
  public get getNia(): Number {
    return this.nia;
  }
  public get getCiclo(): string {
    return this.ciclo;
  }
  public set setNia(numero: Number) {
    this.nia = numero;
  }
}
